Tests/Integration/DSCResources/Firefox.integration.tests.ps1

```diff
@@ -14,18 +14,27 @@ try
 
     foreach ($stig in $stigList)
     {
-        [xml] $powerstigXml = Get-Content -Path $stig.Path
+        $orgSettingsPath = $stig.Path.Replace('.xml', '.org.default.xml')
+        $blankSkipRuleId = Get-BlankOrgSettingRuleId -OrgSettingPath $orgSettingsPath
+        $powerstigXml = [xml](Get-Content -Path $stig.Path) |
+            Remove-DscResourceEqualsNone |
+            Remove-SkipRuleBlankOrgSetting -OrgSettingPath $orgSettingsPath
```

Comment:
Indent here and any of the other tests that need it

Reply:
Done